jobs/filings-notebook-report/daily/incorpfilings.ipynb

```diff
@@ -200,7 +201,26 @@
     }
    },
    "outputs": [],
-   "source": []
+   "source": [
+    "%%sql stat_filings_total_completed  <<\n",
+    "SELECT count(*)\n",
+    "FROM businesses b, filings f, users u \n",
```

Comment:
Is this supposed to count the number of Incorp App, or just all filings?
You can also grab the main filing type too, if you wanted to breakdown by Incorps, AR, COA, COD, etc.

Reply:
As required by Kaine, he needs to have the total amount of Incorps. Please see emails communication between Kaine and me:



Yes so we will know the total amount😊

Kaine Sparks
Director, Digital Operations|  BC Registries and Online Services
Service BC 
Ministry of Citizens’ Services
T: 250-415-3957 | Web: http://www.servicebc.gov.bc.ca 
“Access to government services made easy”

From: Chen, Steven CITZ:EX <Steven.Chen@gov.bc.ca> 
Sent: August 5, 2020 8:43 AM
To: Sparks, Kaine CITZ:EX <Kaine.Sparks@gov.bc.ca>
Subject: RE: WFH

Good morning Kaine, did you mean to add total amount of incorps to date for benefit incorporation to the report attached?

Thanks,

-----------------
Steven Chen

Service BC
Ministry of Technology, Innovation & Citizens’ Services
Web: http://www.servicebc.gov.bc.ca
“Access to government services made easy”

From: Sparks, Kaine CITZ:EX <Kaine.Sparks@gov.bc.ca> 
Sent: August 5, 2020 8:36 AM
To: Chen, Steven CITZ:EX <Steven.Chen@gov.bc.ca>
Subject: RE: WFH

Hi Steven, 

Do you have stuff to work on? Checking if we can change the benefit incorporation report to having the total amount of incorps to date?

Kaine Sparks
Director, Digital Operations|  BC Registries and Online Services
Service BC 
Ministry of Citizens’ Services
T: 250-415-3957 | Web: http://www.servicebc.gov.bc.ca 
“Access to government services made easy”

Reply:
What do you mean by "Incorps"?
If you want the number of IncorporattionApplications, you'd need to add that criteria in.
if you want the total number of Benefit Corporations, you can just count off the business table.
I'm not sure what joining those tables together will give you?
It would pick up all filings (Incorporations, Alterations, COA, COD, etc)